fmn/rules/utils.py

```diff
@@ -67,6 +72,32 @@ def get_fas(config):
     return _FAS
 
 
+def _paginate_pagure_data(url, params, key):
```

Comment:
No, the test coverage for the diff needs to be 100%. I've refactored this PR locally to make the API more sensible and written most of the tests, but the staging Pagure instance seems to have issues with packages with groups as committers or something: e.g. https://src.stg.fedoraproject.org/pagure/api/0/rpms/python-requests. I can't finish the tests without this working.